src/app/ubs-admin/components/ubs-admin-employee/employee-form/employee-form.component.html

```diff
@@ -0,0 +1,61 @@
+<h1>Додати працівника</h1>
+<form [formGroup]="employeeForm" (ngSubmit)="submit()">
+  <div class="form-group">
+    <h6>*Обов'язкові поля</h6>
+  </div>
+  <div class="form-group">
+    <label for="name">Ім'я*</label>
+    <input formControlName="firstName" placeholder="Placeholder" />
+  </div>
+  <div class="form-group">
+    <label>Прізвище*</label>
+    <input formControlName="lastName" placeholder="Placeholder" />
+  </div>
+  <div class="form-group">
+    <label>Номер телефону*</label>
+    <input formControlName="phoneNumber" placeholder="+380" />
+  </div>
+  <div class="form-group">
+    <label>Email</label>
+    <input formControlName="email" placeholder="" />
+  </div>
+  <label>Посада*</label>
+  <div class="checkbox-section-role">
+    <div *ngFor="let role of roles">
+      <label class="checkbox-label"
+        >{{ role.name }}<input class="checkbox-input" type="checkbox" (change)="onCheckChangeRole({ id: role.id, name: role.name })"
+      /></label>
+    </div>
+  </div>
+  <label>Станція приймання*</label>
+  <div class="checkbox-section-stantion">
+    <div *ngFor="let location of locations">
+      <label class="checkbox-label-station"
+        >{{ location.name
+        }}<input class="checkbox-input" type="checkbox" (change)="onCheckChangeLocation({ id: location.id, name: location.name })"
+      /></label>
+    </div>
+  </div>
+  <label class="photo-label">Фото</label>
+  <div class="custom-file-upload">
+    <div *ngIf="filePath && filePath !== ''">
+      <img [src]="filePath" [alt]="employeeForm.value.name" />
+    </div>
+    <input formControlName="image" class="upload-file" type="file" accept="image/*" (change)="imagePreview($event)" #fileUpload />
+    <div class="image-name" *ngIf="uploaded">
+      <p class="img-text">{{ this.selectedFile.name }}</p>
+      <button class="reset-image" (click)="fileUpload.click()">
+        <svg xmlns="http://www.w3.org/2000/svg" width="11" height="11" fill="currentColor" class="bi bi-x" viewBox="0 0 16 16">
+          <path
+            d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"
+          />
+        </svg>
+      </button>
+    </div>
+    <div class="upload-text" *ngIf="!uploaded">Перетягніть зображення, або <a (click)="fileUpload.click()">завантажте з пристрою</a></div>
+  </div>
+  <div class="button-row">
+    <button mat-stroked-button mat-dialog-close color="#13aa57">Скасувати</button>
+    <button mat-raised-button color="#13aa57" type="submit">Додати</button>
```

Comment:
can we use class instead of color="#13aa57" ?

Reply:
created class for this button